### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by schneider-clarisse**
First couple attempts at restyling, including a new side nav and better conference switcher modal

### CODE REVIEW

File: src/js/pages/attendee/Layout.jsx [hunk 1]
```diff
@@ -1,25 +1,151 @@
...
+    ConferenceStore.on('loadedConferences', this.loadConferences);
+    ConferenceStore.on('error', this.showError);
+    BrandStore.on('receivedBranding', this.setBranding);
+    BrandStore.on('error', this.showError);
+  }
+
+  componentWillUnmount() {
+    PreferenceStore.removeListener('loaded', this.loadPreferences);
+    PreferenceStore.removeListener('error', this.showError);
+    ConferenceStore.removeListener('loadedConferences', this.loadedConferences);
```
Comment (larajanecka) [inline src/js/pages/attendee/Layout.jsx:38]: gotta remove the listeners for the brand store as well. Memory links are no bueno

File: src/js/pages/attendee/Layout.jsx [hunk 2]
```diff
@@ -1,25 +1,151 @@
...
+      breakPoint: '10000',
+    }
+
+    const items = [
+      <SidebarItem href='/'>Home</SidebarItem>,
+    ];
+    if (newsfeed) {
+      items.push(<SidebarItem href='/newsfeed'>News Feed</SidebarItem>);
+    }
+    Array.prototype.push.apply(items, [
```
Comment (larajanecka) [inline src/js/pages/attendee/Layout.jsx]: why not `items.push(`? Just wondering the difference here.

File: package.json [hunk 3]
```diff
@@ -20,6 +20,9 @@
     "jwt-decode": "2.1.0",
     "moment": "2.17.1",
     "node-sass": "3.13.1",
+    "postcss": "^5.2.16",
+    "postcss-cssnext": "^2.10.0",
+    "postcss-loader": "^1.3.3",
```
Comment (larajanecka) [inline package.json:25]: react-toolbox missing, also might require updating the version of react used

**Review by larajanecka — CHANGES_REQUESTED** [review]
The organizer app appears to have some problems, the menu doesn't collapse and there's an error in the console `Cannot read property 'insertRule' of undefined
    at BrandStore.Layout._this.setBranding (Layout.jsx:22)`

File: package.json (hunk 3 above)
Comment (schneider-clarisse) [reply package.json:25]: (y)

**Comment by schneider-clarisse** [comment]
Organizer navbar is pinned unless less than a certain width.

File: src/js/pages/attendee/Layout.jsx (hunk 2 above)
Comment (schneider-clarisse) [reply src/js/pages/attendee/Layout.jsx]: good catch

File: src/js/pages/attendee/Layout.jsx (hunk 1 above)
Comment (schneider-clarisse) [reply src/js/pages/attendee/Layout.jsx:38]: gg

**Comment by schneider-clarisse** [comment]
fixes #115 and #100

**Comment by schneider-clarisse** [comment]
@larajanecka plz check when you have a minute

**Review by larajanecka — APPROVED** [review]

**Comment by larajanecka** [comment]
lgtm

**Pull request merged**